source/upgrade.txt

```diff
@@ -141,6 +141,14 @@ The v4.0 driver introduces the following breaking changes to the build system:
   - ``BSONCXX_POLY_USE_SYSTEM_MNMLSTC``.
   - ``BSONCXX_POLY_USE_BOOST``.
 
+- Drops support for the ``MONGOCXX_OVERRIDE_DEFAULT_INSTALL_PREFIX`` configuration
+  option. This option is implicitly set to ``OFF``.
```

Comment:
```suggestion
- Drops support for the ``MONGOCXX_OVERRIDE_DEFAULT_INSTALL_PREFIX`` configuration
  option. Default CMake behavior for `CMAKE_INSTALL_PREFIX` is now respected.
```
Wording tweak to direct users to CMake options and its (default) behavior. I am having trouble suggesting an alternative wording, but the point we want to make is, "we no longer have any potentially-surprising behavior that differs from normal, common CMake behavior concerning the handling of the install prefix path".